doc/FREQUENTLY_MADE_SUGGESTIONS.md

```diff
@@ -140,16 +134,6 @@ Savescumming is not a normal part of the game, and there is no intention of ever
 
 I’m 100% fine having a game mode available where this happens.  However, at least in the current state of development, it’s way too easy to accumulate NPC followers and end up becoming effectively immortal.  Thus trivializes a lot of aspects of the game and encouraging even more reckless behavior so things would need to be done to make it more difficult to acquire NPC followers, or at least NPC followers that you could switch to when you died.
 
-#### Psychic powers: mod only
-
-Not happening, it simply doesn’t fit the theme of the game.
-However, we’d be happy to have support for it in the code so that a mod can have psychic powers.
-
-#### Magic powers: mod only
-
-Not happening, it simply doesn’t fit the theme of the game.
-However, the Magiclysm mod is distributed with the game and is very extensible in JSON to support other systems of magic.
-
```

Comment:
Why remove this? The stance for either hasn't changed in vanilla, and the answer about Magiclysm is still pertinent. Mention MoM the same way in the part about psychic powers.

Reply:
Done.